legend-engine-core/legend-engine-core-language-pure/legend-engine-language-pure-grammar-api/src/test/java/org/finos/legend/engine/language/pure/grammar/api/test/TestJsonToGrammarApi.java

```diff
@@ -288,7 +288,8 @@ public void testUnionModelMapping()
                 "  meta::pure::mapping::modelToModel::test::shared::dest::Person[p1]: Pure\n" +
                 "  {\n" +
                 "    ~src meta::pure::mapping::modelToModel::test::shared::src::_Person\n" +
-                "    ~filter $src.fullName->startsWith('Johny')\n" +
+                "    ~filter \n" +
+                "       $src.fullName->startsWith('Johny')\n" +
```

Comment:
Why has the formatting of filter changed here and below? Isn't this change formatting if statements? I don't see an if statement involved here